Lesson008/Lesson005/Task 3 HW05.SchoolControls/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace Task3HW05.SchoolControls
+{
+   internal class Program
+    {
+        static int YourQuarter(int controlTest, string firstQuarter, string secondQuarter, string thridQuarter, string fourthQuarter)
+        {
+            if (controlTest <= 15)
+            {
+                Console.WriteLine(firstQuarter);
+            }
+            else if (controlTest > 15 & controlTest <= 30)
+            {
+                Console.WriteLine(secondQuarter);
+            }
+            else if (controlTest > 30 & controlTest <= 45)
+            {
+                Console.WriteLine(thridQuarter);
+            }
+            else if (controlTest > 45 & controlTest <= 60)
+            {
+                Console.WriteLine(fourthQuarter);
+            }
+            return controlTest;
+        }
+        static void Main(string[] args)
+        {
+            string firstQuarter = "First Quarter", secondQuarter = "Second Quarter ", thridQuarter = "Thrid Quarter", fourthQuarter = "Fourth Quarter";
+            int controlTest;
+
+            Console.WriteLine("Your control test number: ");
+            controlTest = Convert.ToInt32(Console.ReadLine());
+
+            YourQuarter(controlTest, firstQuarter, secondQuarter, thridQuarter, fourthQuarter);
```

Comment:
You need to pass  controlTest in method, move consts in method and remane him like GetQuarter().

Reply:
I didn't look further, because the same errors. Move on yourself and correct the errors in the remaining tasks relying on the examples and corrections above.

Reply:
Correct